test/journal-route-test.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+const Journal = require('../model/journal.js');
+const PORT = process.env.PORT || 3000;
+
+// process.env.MONGODB_URI = 'mongodb://localhost/journaltest';
+
+require('../server.js');
+
+const url = `http://localhost:${PORT}`;
+const exampleJournal = {
+  name: 'test journal name',
+  subject: 'test journal subject'
+
```

Comment:
Remember to include all of the properties you have in your journalSchema constructor. => timestamp: new Date()